zerver/views/realm.py

```diff
@@ -125,6 +129,12 @@ def update_realm(
     }:
         return json_error(_("Invalid video_chat_provider {}").format(video_chat_provider))
 
+    if oss_showcase_policy is not None and realm.plan_type != Realm.STANDARD_FREE:
+        return json_error(_("Your organization does not qualify to enable this feature."))
+
+    if oss_showcase_policy is not None and settings.OSS_SHOWCASE is False:
+        return json_error(_("The following feature must be enabled on the hosted system."))
```

Comment:
I swapped the order of these checks, and also switched the error message to try to make life a bit easier for translators (avoiding unnecessarily specific strings unique to this widget).